<!DOCTYPE html>
<html>
<head>
  <title>Lorenzo Bini</title>
  <link rel="icon" href="static/images/profile_round.png" type="image/x-icon">
  <link rel="stylesheet" type="text/css" href="styles.css">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
  <meta charset="UTF-8">
  <meta name="google-site-verification" content="xGwNrd6xXdCT0TtOFDd64vEACPbW90IxihTxso76EBM" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>
<body>
  <div class="container">
    <aside>
      <div class="profile-sidebar">
        <img src="static/images/profile_round.png" alt="Profile Picture" class="profile-picture">
        <h1>Lorenzo Bini</h1>
        <p>PhD Candidate in Artificial Intelligence at the University of Geneva.</p>
        <ul class="social-links">
          <li><a href="https://github.com/LorenzoBini4" target="_blank" rel="noopener noreferrer"><i class="fab fa-github"></i></a></li>
          <li><a href="https://scholar.google.com/citations?user=RRB-j2kAAAAJ&hl=it" target="_blank" rel="noopener noreferrer"><i class="fas fa-graduation-cap"></i></a></li>
          <li><a href="https://www.linkedin.com/in/lorenzo-bini4/" target="_blank" rel="noopener noreferrer"><i class="fab fa-linkedin"></i></a></li> 
          <li><a href="https://twitter.com/LorenzoBini47" target="_blank" rel="noopener noreferrer"><i class="fab fa-twitter"></i></a></li> 
          <li><a href="https://viper.unige.ch/doku.php?id=home" target="_blank" rel="noopener noreferrer"><i style="font-size:16px" class="fa">&#xf0c0;</i></a></li>
          <li><a href="https://lorenzobini4.github.io/static/images/ResumeBiniLorenzo.pdf" target="_blank"><strong>CV</strong></a></li>
	</ul>
      </div>
    </aside>
    <main>
      <section id="about">
        <div class="profile">
          <div class="profile-text">
            <p> Greetings everyone, I'm Lorenzo, Ph.D. Candidate in Artificial Intelligence at the <a href="https://www.unige.ch/sciences/en/" target="_blank" rel="noopener noreferrer">University of Geneva</a>. I work with <a href="https://viper.unige.ch/doku.php?id=team:stephane_marchandmaillet" target="_blank" rel="noopener noreferrer">Stéphane Marchand-Maillet</a> 
             as a member of the <a href="https://viper.unige.ch/doku.php?id=about" target="_blank" rel="noopener noreferrer">VIPER</a> group, which closely collaborates with the <a href="http://dmml.ch/" target="_blank" rel="noopener noreferrer">DMML</a> group.  My research is focused on developing self-supervised (SSL) and active learning (AL) strategies to streamline the costly process of acquiring labels in high-dimensional data tasks. 
	     Specializing in Graph Neural Networks (GNNs), my work ranges from graph adversarial learning to representation learning, particularly on anomaly detection tasks in self or weakly supervised contexts. 
	     I am also developing generative models (e.g., DDPMs) for 3D genomics, flow cytometry, scRNA, spatial transcriptomics, and multi-omics data. Lately, I've been focusing on using LLMs to streamline routine hospital processes. 
	     <br><br>In collaboration with the Geneva University Hospital (<a href="https://www.hug.ch/en/hematologie/research" target="_blank" rel="noopener noreferrer">HUG</a>), I'm currently working on detecting Minimal Residual Disease (MRD) of Acute Lymphoblastic and Myeloid Leukemia 
             from Flow Cytometry data, where SSL and AL methods help reduce the burden of physician-led annotation by enhancing data quality and efficiency. In this scenario, it is crucial to develop efficient training and inference models for single-cell classification.
	     Additionally, I'm also involved as a teaching assistant for the “Introduction to Computational Finance”, “Natural Language Processing” and "Information Retrieval" courses at the <a href="https://cui.unige.ch/en/" target="_blank" rel="noopener noreferrer">CUI</a>.
		 <br><br> Recently, I’ve joined <a href="https://www.cs.cmu.edu/~jianma/" target="_blank" rel="noopener noreferrer">Jian Ma</a>’s group at the <a href="https://cbd.cmu.edu/" target="_blank" rel="noopener noreferrer">Ray and Stephanie Lane Computational Biology Department</a> at <a href="https://www.cmu.edu/" target="_blank" rel="noopener noreferrer">Carnegie Mellon University</a> 
				for a 6-month internship, where I’ll be working on GenAI for epigenomics and gene-regulation and a stack of other cool comp-bio projects. Excited to see what we'll build together! </p>
            <p>Check out our latest works:</p>
<ul>
    <li>The first comprehensive <a href="https://github.com/LorenzoBini4/FlowCyt-Classification-Benchmark" target="_blank" rel="noopener noreferrer">benchmark</a> for multi-class single-cell classification on flow cytometry data, using GNNs and many other state-of-the-art single-cell deep learning techniques!</li>
    <li>A novel method to inject biological priors on different state-of-the-art GNNs for hierarchical single-cell classification; our plug-in module <a href="https://openreview.net/forum?id=HJ5ILdCHFx" target="_blank" rel="noopener noreferrer">FCHC-GNN</a>!</li>
    <li><a href="https://arxiv.org/abs/2409.03463" target="_blank" rel="noopener noreferrer">Decoding attention</a> for domain-dependent interpretable GNNs: the first study revealing the emergence of <a href="https://arxiv.org/pdf/2409.03463" target="_blank" rel="noopener noreferrer">Massive Activations (MAs)</a> within GNNs attention mechanisms.
	    By rigorously comparing activation distributions between untrained (base) and fully trained models across architectures such as GraphTransformer, GraphiT, and SAN (and on diverse benchmarks like ZINC, TOX21, and OGBN-PROTEINS) we reveal how MAs emerge, 
	    their direct link to learned biases, and their impact on model robustness. Furthermore, our work introduces a novel detection framework based on activation ratio distributions and proposes the Explicit Bias Term (EBT) as an effective countermeasure to turn MAs off!  
    </li>
    <li>	
	<a href="https://arxiv.org/abs/2506.13344" target="_blank" rel="noopener noreferrer">LapDDDPM</a>: a novel conditional Graph Diffusion Probabilistic Model for robust and high-fidelity scRNA-seq generation! 
	    LapDDPM uniquely integrates graph-based representations with a score-based diffusion model, enhanced by a novel spectral adversarial perturbation mechanism on graph edge weights. Here's how: we leverage Laplacian Positional Encodings 
	    (LPEs) to enrich the latent space with crucial cellular relationship information; we develop a conditional score-based diffusion model for effective learning and generation from complex scRNA-seq distributions; and we employ a unique spectral adversarial training scheme on graph edge weights, boosting robustness against structural variations!
     </li>
     <li> Presenting a new state-of-the-arte <a href="https://arxiv.org/abs/2506.20362" target="_blank" rel="noopener noreferrer">self-supervised graph learning</a> framework that bypasses the need for negative sampling by leveraging spectral bootstrapping techniques! 
	Our method integrates Laplacian-based signals into the learning process, allowing the model to effectively capture rich structural representations without relying on contrastive objectives or handcrafted augmentations. 
	We precompute spectral augmentations through max-min centrality-guided optimization, enabling structural supervision, and integrate an adversarial bootstrapped training scheme that further strengthens feature learning and robustness!
     </li>
		<li> Our <a href="https://arxiv.org/pdf/2409.03463" target="_blank" rel="noopener noreferrer">MAs</a> paper has been accepted as oral presentation at the <a href="https://ecai2025.org" target="_blank" rel="noopener noreferrer">European Conference on Artificial Intelligence (ECAI’25), Bologna</a>!
		</li> </ul>
        </p></div>
        </div>
      </section>

      <section id="publications">
        <h2>My Selected Papers</h2>
        <div class="publicationcontainer">
          <div>
            <div class="entrytitle">
		<a href="https://arxiv.org/abs/2506.13344" target="_blank" rel="noopener noreferrer" class="invisiblelink">
                LapDDPM: A Conditional Graph Diffusion Model for scRNA-seq Generation with Spectral Adversarial Perturbations
              </a>
            </div>
            <div class="entrysubtitle">
              Lorenzo Bini, Stéphane Marchand-Maillet
            </div>
            <div class="entrysubsubtitle">
	     ICML'2025 + GenBio Workshop: The Second Workshop on Generative AI and Biology, Vancouver. 
            </div>
          </div>
          <div class="publicationdescription">
	    <a href="https://arxiv.org/pdf/2506.13344" target="_blank" rel="noopener noreferrer">
              <i class="fa fa-file-alt" aria-hidden="true"></i>&nbsp;Paper
            </a>
            &ensp;			  
            <a href="https://github.com/LorenzoBini4/laplace-DDPM" target="_blank" rel="noopener noreferrer">
              <i class="fab fa-fw fa-github" aria-hidden="true"></i>Code
            </a>
            &ensp;
	    <a href="citations/lapddpm-icml.html" target="_blank" rel="noopener noreferrer">
              <i class="fa fa-xs fa-quote-right" aria-hidden="true"></i>&nbsp;Cite					    
            </a>
          </div>
        </div>    
	      
	<div class="publicationcontainer">
          <div>
            <div class="entrytitle">
		<a href="https://arxiv.org/abs/2506.20362" target="_blank" rel="noopener noreferrer" class="invisiblelink">	
                Self-Supervised Graph Learning via Spectral Bootstrapping and Laplacian-Based Augmentations
              </a>
            </div>
            <div class="entrysubtitle">
              Lorenzo Bini, Stéphane Marchand-Maillet
            </div>
            <div class="entrysubsubtitle">
             To appear in 2025 (preprint, under double-blind review as a conference paper).
            </div>
          </div>
          <div class="publicationdescription">
            <a href="https://arxiv.org/pdf/2506.20362" target="_blank" rel="noopener noreferrer">
              <i class="fa fa-file-alt" aria-hidden="true"></i>&nbsp;Paper
            </a>
            &ensp;
            <a href="https://github.com/LorenzoBini4/laplaceGNN" target="_blank" rel="noopener noreferrer">
              <i class="fab fa-fw fa-github" aria-hidden="true"></i>Code
            </a>
            &ensp;
	    <a href="citations/laplaceGNN.html" target="_blank" rel="noopener noreferrer">
              <i class="fa fa-xs fa-quote-right" aria-hidden="true"></i>&nbsp;Cite
            </a>
          </div>
        </div>    
	      
        <div class="publicationcontainer">
          <div>
            <div class="entrytitle">
              <a href="https://arxiv.org/abs/2409.03463" target="_blank" rel="noopener noreferrer" class="invisiblelink">
                Massive Activations in Graph Neural Networks: Decoding Attention for Domain-Dependent Interpretability
              </a>
            </div>
            <div class="entrysubtitle">
              Lorenzo Bini*,  Marco Sorbi*, Stéphane Marchand-Maillet
            </div>
            <div class="entrysubsubtitle">
             European Conference on Artificial Intelligence (ECAI'2025), Bologna - Oral Presentation + ICLR'2025 Workshop XAI4Science: From Understanding Model Behavior to Discovering New Scientific Knowledge, Singapore.
            </div>
          </div>
          <div class="publicationdescription">
            <a href="https://arxiv.org/pdf/2409.03463" target="_blank" rel="noopener noreferrer">
              <i class="fa fa-file-alt" aria-hidden="true"></i>&nbsp;Paper
            </a>
            &ensp;
            <a href="https://github.com/LorenzoBini4/GNN-MAs" target="_blank" rel="noopener noreferrer">
              <i class="fab fa-fw fa-github" aria-hidden="true"></i>Code
            </a>
            &ensp;
            <a href="citations/gnn-ma.html" target="_blank" rel="noopener noreferrer">
              <i class="fa fa-xs fa-quote-right" aria-hidden="true"></i>&nbsp;Cite
            </a>
          </div>
        </div>
       
        <div class="publicationcontainer">
          <div>
            <div class="entrytitle">
              <a href="https://icml.cc/virtual/2024/35934" target="_blank" rel="noopener noreferrer" class="invisiblelink">
                Injecting Hierarchical Biological Priors into Graph Neural Networks for Flow Cytometry Prediction
              </a>
            </div>
            <div class="entrysubtitle">
               Lorenzo Bini, Stéphane Marchand-Maillet 	
            </div>
            <div class="entrysubsubtitle">
               ICML'2024 + Workshop on Accessible and Efficient Foundation Models for Biological Discovery, Wien, Austria. 
            </div>
          </div>
          <div class="publicationdescription">
            <a href="https://openreview.net/pdf?id=HJ5ILdCHFx" target="_blank" rel="noopener noreferrer">
              <i class="fa fa-file-alt" aria-hidden="true"></i>&nbsp;Paper
            </a>
            &ensp;
            <a href="https://github.com/LorenzoBini4/FCHC-GNN-Hierarchical" target="_blank" rel="noopener noreferrer">
              <i class="fab fa-fw fa-github" aria-hidden="true"></i>Code
            </a>
            &ensp;
            <a href="citations/FCHC-GNNs.html" target="_blank" rel="noopener noreferrer">
              <i class="fa fa-xs fa-quote-right" aria-hidden="true"></i>&nbsp;Cite
            </a>
          </div>
        </div>

      <div class="publicationcontainer">
          <div>
            <div class="entrytitle">
              <a href="https://proceedings.mlr.press/v248/bini24a.html" target="_blank" rel="noopener noreferrer" class="invisiblelink">
                FlowCyt: A Comparative Study of Deep Learning Approaches for Multi-Class Classification in Flow Cytometry Benchmarking
              </a>
            </div>
            <div class="entrysubtitle">
              Lorenzo Bini, Margarita Liarou, Thomas Matthes, Stéphane Marchand-Maillet
            </div>
            <div class="entrysubsubtitle">
              Conference on Health, Inference, and Learning (CHIL'24), New-York, NY.
            </div>
          </div>
          <div class="publicationdescription">
            <a href="https://proceedings.mlr.press/v248/bini24a.html" target="_blank" rel="noopener noreferrer">
              <i class="fa fa-file-alt" aria-hidden="true"></i>&nbsp;Paper
            </a>
            &ensp;
            <a href="https://github.com/LorenzoBini4/FlowCyt-Classification-Benchmark" target="_blank" rel="noopener noreferrer">
              <i class="fab fa-fw fa-github" aria-hidden="true"></i>Code
            </a>
            &ensp;
            <a href="citations/flowcyt-benchmark-deep-learning.html" target="_blank" rel="noopener noreferrer">
              <i class="fa fa-xs fa-quote-right" aria-hidden="true"></i>&nbsp;Cite
            </a>
            &ensp;
            <a href="https://cuicloud.unige.ch/index.php/s/55PHBLEynrp5pN8" target="_blank" rel="noopener noreferrer">
              <i class="fas fa-cloud-download-alt" aria-hidden="true"></i>&nbsp;Data
            </a>
          </div>
        </div>
       </section>
	      
    <!--  <div class="publicationcontainer">
	  <div>
	    <div class="entrytitle">
	      <a href="https://arxiv.org/abs/2402.18611" target="_blank" rel="noopener noreferrer" class="invisiblelink">
		HemaGraph: Breaking Barriers in Hematologic Single Cell Classification with Graph Attention
	      </a>
	    </div>
	    <div class="entrysubtitle">
	      Lorenzo Bini*,  Fatemeh Nassajian Mojarrad*, Thomas Matthes, Stéphane Marchand-Maillet
	    </div>
	    <div class="entrysubsubtitle">
	     arXiv 2024 (pre-print)
	    </div>
	  </div>
	  <div class="publicationdescription">
	    <a href="https://arxiv.org/pdf/2402.18611.pdf" target="_blank" rel="noopener noreferrer">
	      <i class="fa fa-file-alt" aria-hidden="true"></i>&nbsp;Paper
	    </a>
	    &ensp;
	    <a href="https://github.com/LorenzoBini4/GNN-HemaGraph" target="_blank" rel="noopener noreferrer">
	      <i class="fab fa-fw fa-github" aria-hidden="true"></i>Code
	    </a>
	    &ensp;
	    <a href="citations/hemagraph-breaking-barriers.html" target="_blank" rel="noopener noreferrer">
	      <i class="fa fa-xs fa-quote-right" aria-hidden="true"></i>&nbsp;Cite
	    </a>
	  </div>
	</div> -->
  
<!-- 	  <div class="publicationcontainer">
          <div>
            <div class="entrytitle">
              <a href="https://arxiv.org/abs/2402.18610" target="_blank" rel="noopener noreferrer" class="invisiblelink">
                Why Attention Graphs Are All We Need: Pioneering Hierarchical Classification of Hematologic Cell Populations with LeukoGraph
              </a>
            </div>
            <div class="entrysubtitle">
               Fatemeh Nassajian Mojarrad*, Lorenzo Bini*, Thomas Matthes, Stéphane Marchand-Maillet 	
            </div>
            <div class="entrysubsubtitle">
               arXiv 2024 (pre-print)
            </div>
          </div>
          <div class="publicationdescription">
            <a href="https://arxiv.org/pdf/2402.18610.pdf" target="_blank" rel="noopener noreferrer">
              <i class="fa fa-file-alt" aria-hidden="true"></i>&nbsp;Paper
            </a>
            &ensp;
            <a href="https://github.com/LorenzoBini4/Attention-LeukoGraph" target="_blank" rel="noopener noreferrer">
              <i class="fab fa-fw fa-github" aria-hidden="true"></i>Code
            </a>
            &ensp;
            <a href="citations/why-attention-graphs-are-all-we-need.html" target="_blank" rel="noopener noreferrer">
              <i class="fa fa-xs fa-quote-right" aria-hidden="true"></i>&nbsp;Cite
            </a>
          </div>
        </div -->

	<section id="Publications List">
        <h2>Publications List</h2>
        <p>See the <a href="https://scholar.google.com/citations?user=RRB-j2kAAAAJ&hl=it" target="_blank" rel="noopener noreferrer">publications list</a> for a more complete report.</p>
      </section>
      
      <section id="Contact Me">
        <h2>Contact Me</h2>
        <!--<p>Email: <a href="mailto:lorenzo dot bini at unige dot ch" target="_blank" rel="noopener noreferrer"">lorenzo.bini@unige.ch</a></p>-->
	<p>Email: lorenzo dot bini at unige dot ch</p>     
        <p>Geneva-based: Department of Computer Science, BAT A, <a href="https://maps.app.goo.gl/Qsgwuzun3JRMc2tZ7?g_st=ic" target="_blank" rel="noopener noreferrer">Route de Drize 7 1227 Carouge, Switzerland</a>, Office Room #221</p>
      </section>
    </main>
  </div>
</body>
</html>






